src/components/ServiceMessage.tsx

```diff
@@ -1,26 +1,40 @@
 import { FC } from "react"
 
 import { ServiceMessage as ServiceMessageType } from "../api/ServiceMessage"
-import { cl, getTimestamp } from "../utils"
+import { NBSP, cl, dateFormat } from "../utils"
 
 import { Header3 } from "./text"
 
-
 type ServiceMessageProps = {
   serviceMessage: ServiceMessageType
   className?: string
 }
 
-const ServiceMessage: FC<ServiceMessageProps> = ({ serviceMessage: { content, resolution_content, created_at, resolved, resolved_at, door }, className }) => (
-  <div className={cl("relative space-y-4", className)}>
-    <Header3>
-      {resolved
-        ? (door ? `Feil på luke ${door} (rettet)` : "Feil (rettet)")
-        : (door ? `Feil på luke ${door}` : "Feil")
-      }
-    </Header3>
-    <time className="absolute top-2 right-6 text-xs">{getTimestamp(created_at)}{resolved_at && ` — ${getTimestamp(resolved_at)}`}</time>
-
+const ServiceMessage: FC<ServiceMessageProps> = ({
+  serviceMessage: { content, resolution_content, created_at, resolved, resolved_at, door },
+  className
+}) => (
+  <div
+    className={cl(
+      "w-full flex flex-col gap-8 border-2 p-8 rounded-md",
+      resolved ? "border-yellow-400" : "border-red-700",
+      className
+    )}
+  >
+    <div className="flex justify-between flex-wrap">
+      <Header3 as="h2">
+        {`Feil${door ? ` på luke ${door}` : ""}${resolved ? " (rettet)" : ""}`}
+      </Header3>
+      <span className="text-xs">
+        <time>{dateFormat(new Date(created_at), "short")}</time>
+        {resolved_at && (
+          <>
+            {`${NBSP}—${NBSP}`}
```

Comment:
Spørsmål heller enn kritikk her: Er det kanskje mer ryddig å wrappe i et span med nowrap heller enn å smelle inn nbsp?

For såvidt litt tungvint å wrappe en template string på denne måten i JSX.

Reply:
Ja, det er nok ryddigere. I fix